doc/release_notes_1.1.rst

```diff
@@ -0,0 +1,514 @@
+.. _release_notes_1.1:
+
+ACRN v1.1 (June 2019)
+#####################
+
+We are pleased to announce the release of ACRN version 1.1.
+
+ACRN is a flexible, lightweight reference hypervisor, built with
+real-time and safety-criticality in mind, optimized to streamline embedded
+development through an open source platform. Check out the :ref:`introduction` for more information.
+All project ACRN source code is maintained in the https://github.com/projectacrn/acrn-hypervisor
+repository and includes folders for the ACRN hypervisor, the ACRN device
+model, tools, and documentation. You can either download this source code as
+a zip or tar.gz file (see the `ACRN v1.1 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v1.1>`_)
+or use Git clone and checkout commands::
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v1.1
+
+The project's online technical documentation is also tagged to correspond
+with a specific release: generated v1.1 documents can be found at https://projectacrn.github.io/1.1/.
+Documentation for the latest (master) branch is found at https://projectacrn.github.io/latest/.
+ACRN v1.1 requires Clear Linux* OS version 29970 or newer. Please follow the
+instructions in the :ref:`getting-started-apl-nuc`.
+
+Version 1.1 major features
+**************************
+
+Hybrid Mode Introduced
+======================
+In hybrid mode, a Zephyr OS is launched by the hypervisor even before the Service OS is
+launched (pre-launched), with dedicated resources to achieve highest level of isolation.
+This is designed to meet the needs of a FuSa certifiable safety OS.
+
+More types of Operating systems supported
+=========================================
+* The `Zephyr RTOS <https://zephyrproject.org>`_ can be a pre-launched Safety OS in hybrid mode.
+  It can also be a post-launched (launched by Service OS, not the hypervisor) as a guest OS.
+* VxWorks as a post-launched RTOS for industrial usages.
+* Windows as a post-launched OS
+
+Document updates
+================
+We have many `reference documents available <https://projectacrn.github.io>`_, including:
+
+* :ref:`Update: Using PREEMPT_RT-Linux for real-time UOS <rt_linux_setup>`
+* :ref:`How to enable OVS in ACRN <open_vswitch>`
+* :ref:`Enable QoS based on runC container <acrn-dm_qos>`
+* :ref:`Using partition mode on NUC <using_partition_mode_on_nuc>`
```

Comment:
We should also list the tutorials on Zephyr and VxWorks.

I realize the the Zephyr tutorial already existed in 1.0 but since we talked about new Guest OS support, we might as well list it here again.

Reply:
It's unfortunate that we do not have a tutorial on WaaG, the code is merged and unless it's not functional yet, we should document it.

Reply:
@gvancuts , We are partially enabled WaaG now, will document it once it is ready.